operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -602,7 +602,15 @@ message GlobalConfig {
   // Currently, two options are supported: "third-party-jwt" and "first-party-jwt".
   string jwtPolicy = 57;
 
-  // The next available key is 58
+
+  // Configure whether provisions a certificate to Prometheus through Istio Agent.
+  // When this option is set as true, a sidecar is deployed along Prometheus to
+  // provision a certificate through Istio Agent to Prometheus. The provisioned certificate
+  // is shared with Prometheus through mounted files.
+  // When this option is set as false, this certificate provisioning mechanism is disabled.
+  google.protobuf.BoolValue  provisionPrometheusCert = 58;
```

Comment:
If this is prometheus config shouldn't it be under prom settings, not global?

Reply:
Done: moved to be under prometheus settings.